doc/source/working_with_kiwi/xml_description.rst

```diff
@@ -757,6 +759,30 @@ configured via the following attributes:
 We will describe the different child elements of `packages` in the following
 sections.
 
+If you want to install patterns, use the `namedCollection` element:
```

Comment:
This should be generalized. `namedCollections` is how you install "groups" in RH/Fedora and "patterns" in SUSE, so this should probably be reworded to account for that.

Reply:
yes I did that when I added the information to the current xml_description_doc_update branch. Thanks